__test__/locale.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, it, expect } from 'vitest'
+import { resolve } from 'node:path'
+import { readdirSync } from 'node:fs'
+import en from '../locales/en-US.json'
+
+function getKeys(obj: any, path = '', result: string[] = []) {
+  for (let key in obj) {
+    if (typeof obj[key] === 'object') {
+      getKeys(obj[key], path ? `${path}.${key}` : key, result);
+    } else {
+      result.push(path ? `${path}.${key}` : key);
+    }
+  }
+  return result;
+}
+
+const locales = readdirSync(resolve(__dirname, '../locales')).filter((file) => {
+  return file.endsWith('.json')
+})
+const defaultKeys = getKeys(en);
+
+describe("should include all keys", () => {
+  locales.forEach((locale) => {
+    it.runIf(!locale.startsWith("en-US"))(`for ${locale}`, () => {
```

Comment:
instead of filtering en-US here, I would remove it earlier and just use `it` here:
```ts
const localesOtherThanEnglish = readdirSync(resolve(__dirname, '../locales')).filter((file) => {
  return file.endsWith('.json') && file !== 'en-US.json'
})
```